doc/devel/contributing.rst

```diff
@@ -455,6 +455,30 @@ Then they will receive messages like
    DEBUG:matplotlib.yourmodulename:Here is some information
    DEBUG:matplotlib.yourmodulename:Here is some more detailed information
 
+**NOTE:** Please avoid using pre-computed stirngs(``f-strings``, ``str.format``,etc.) for logging. There are security & performance
```

Comment:
```suggestion
**NOTE:** Please avoid using pre-computed strings(``f-strings``, ``str.format``,etc.) for logging. There are security and performance
```
There is also some trailing white space it seems, although not clear where.

(I've restarted the doc-build as it looked like there was an intermittent error there, so hopefully it works this time.)